mne/bem.py

```diff
@@ -1355,3 +1357,260 @@ def write_bem_solution(fname, bem):
                                bem['solution'])
         end_block(fid, FIFF.FIFFB_BEM)
         end_file(fid)
+
+
+# #############################################################################
+# Create 3-Layers BEM model from Flash MRI images
+
+def make_flash_bem(subject, subjects_dir, noflash30=False, noconvert=False,
+                   unwarp=False, show=False):
+    """Create 3-Layers BEM model from Flash MRI images
+
+    Parameters
+    -----------
+    subject : string
+        Subject name.
+    subjects_dir : string
+        Directory containing subjects data (Freesurfer SUBJECTS_DIR).
+    noflash30 : bool
+        Skip the 30-degree flip angle data.
+    noconvert : bool
+        Assume that the Flash MRI images have already been converted
+        to mgz files.
+    show : bool
+        Show surfaces in 3D to visually inspect all three BEM
+        surfaces (recommended).
+
+    Notes
+    -----
+    This program assumes that both Freesurfer/FSL, and MNE, including MNE's
+    Matlab Toolbox, are installed properly.
```

Comment:
I think this statement would be better like this:
This program assumes that FreeSurfer and MNE are installed and sourced properly.

The comment about MNE's MATLAB toolbox should refer to FreeSurfer's MATLAB toolbox instead, and I believe it belongs with a description of the unwarp parameter (currently missing)